### PR TITLE
Feature/new gate icons

### PR DESCRIPTION
**Pull request by boland25**
Added new Gate Icons

and I have pycharm working with the flask server but I added some __init__.py files to the diff directories

### CODE REVIEW

File: QPong-Python-Server/flask_server/server.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 from copy import deepcopy
 
 app = Flask(__name__)
-
+print("HI")
```
Comment (boland25) [inline QPong-Python-Server/flask_server/server.py]: i should remove this print line before i merge

File: QPong-Python-Server/flask_server/server.py [hunk 2]
```diff
@@ -129,6 +129,10 @@ def do_measurement():
 
     return str(state_in_decimal)
 
+def run(outsideapp):
+   outsideapp.run(host='127.0.0.1', port=8008)
```
Comment (boland25) [inline QPong-Python-Server/flask_server/server.py]: also, I don't need this method anymore, can remove

File: QPong-Unity-Version/Assets/Scripts/CircuitGridClient.cs [hunk 3]
```diff
@@ -50,6 +50,7 @@ IEnumerator getStatevector(string gateArrayString)
         Debug.Log("Send Gate Array: "+ gateArrayString);
         List<IMultipartFormSection> formData = new List<IMultipartFormSection>();
         formData.Add(new MultipartFormDataSection("gate_array", gateArrayString));
+        Debug.Log("array " + gateArrayString);
```
Comment (boland25) [inline QPong-Unity-Version/Assets/Scripts/CircuitGridClient.cs]: will remove this as well

**Comment by boland25** [comment]
I could never play right with them as w s a d because all the keys are crushed together forcing me to use 1 hand.  But i had meant to put this back to the way it was, i forgot.  I will change it back.

File: QPong-Unity-Version/Assets/Scripts/CircuitGridControl.cs [hunk 4]
```diff
@@ -160,15 +160,15 @@ void Update()
                 {
                     int gate_index = i * columnMax + j;
                     if (gateArray[gate_index] == "I"){
-                        gateObjectArray[gate_index].GetComponent<SpriteRenderer>().sprite = emptyGateSprite;
+                        gateObjectArray[gate_index].GetComponent<Gate>().SetGateIcon(emptyGateSprite);
```
Comment (HuangJunye) [inline QPong-Unity-Version/Assets/Scripts/CircuitGridControl.cs]: This is very neat! I really had no idea how to call a method in Unity. Now I learn it. :)

File: QPong-Unity-Version/Assets/Prefabs/Gate.prefab [hunk 5]
```diff
@@ -0,0 +1,207 @@
+%YAML 1.1
```
Comment (HuangJunye) [inline QPong-Unity-Version/Assets/Prefabs/Gate.prefab]: I didn't know prefab is used in this way. I put the prefabs on scenes... I will do the same for the paddle.

**Pull request merged**